tests/functional/cli/admin/test_item_locate.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+from oio.common.utils import cid_from_name
+from tests.functional.cli import CliTestCase, CommandFailed
+from tests.utils import random_str
+
+
+class ItemLocateTest(CliTestCase):
+    """Functional tests for openio-admin <item> locate."""
+
+    def setUp(self):
+        super(ItemLocateTest, self).setUp()
+
+    def tearDown(self):
+        super(ItemLocateTest, self).tearDown()
+
+    def test_account_locate(self):
+        account = random_str(10)
+        opts = self.get_format_opts()
+        output = self.openio('account create %s %s' % (account, opts))
+        self.assertOutput('%s True' % account, output.strip())
+        output = self.openio_admin('account locate %s %s' % (account, opts))
+        output = output.split(' ')
+        self.assertOutput('account', output[0])
+        self.assertOutput(account, output[1])
+        self.assertOutput('up=True,', output[5])
+        self.assertOutput('None', output[7].strip())
+
+    def _recover_commandfailed(self, func, *args):
+        try:
+            func(*args)
+        except CommandFailed as cf:
+            self.stdout = cf.stdout
+            self.stderr = cf.stderr
+            self.rc = cf.returncode
+            raise
+
+    def test_account_not_found(self):
+        account = random_str(10)
+        opts = self.get_format_opts()
+        self.assertRaises(
+            CommandFailed,
+            self._recover_commandfailed,
+            *[self.openio_admin, 'account locate {} {}'.format(account, opts)])
+        self.assertEquals(1, self.rc)
```

Comment:
`assertEquals` is deprecated. In the future, please use `assertEqual` (without the final 's').